### PR TITLE
Scipy minimiser updated

### PR DESCRIPTION
**Pull request by samwaseda**

### CODE REVIEW

**Comment by samwaseda** [comment]
I merge it now because 1, there are only technical changes; 2, no one is probably using the class; 3, the changes are super local.

File: pyiron/interactive/scipy_minimizer.py [hunk 1]
```diff
@@ -52,13 +40,6 @@ def set_input_to_read_only(self):
     def write_input(self):
         pass
 
-    def interactive_close(self):
```
Comment (jan-janssen) [inline pyiron/interactive/scipy_minimizer.py:55]: Why is this part removed?

Comment (samwaseda) [reply pyiron/interactive/scipy_minimizer.py:55]: It didn't change anything with or without this part.

Comment (jan-janssen) [reply pyiron/interactive/scipy_minimizer.py:55]: Is the output in the HDF5 file the same? in particular in the generic section?

Comment (samwaseda) [reply pyiron/interactive/scipy_minimizer.py:55]: yes

Comment (samwaseda) [reply pyiron/interactive/scipy_minimizer.py:55]: Actually there was anyway nothing in `generic`

Comment (jan-janssen) [reply pyiron/interactive/scipy_minimizer.py:55]: Ok, it seems like `interactive_is_activated()` is not even defined so this was never working ... Still the intention was that the minimiser running in interactive mode could minimize multiple structures which are given to it one by one. But I guess that is a separate pull request.
